arch/posix/include/asm_inline.h

```diff
@@ -0,0 +1,24 @@
+/* Inline assembler kernel functions and macros */
+
+/*
+ * Copyright (c) 2015, Wind River Systems, Inc.
+ * Copyright (c) 2017, Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _ASM_INLINE_H
+#define _ASM_INLINE_H
+
+#if !defined(CONFIG_ARCH_POSIX)
+#error The arch/posix/include/asm_inline.h is only for the POSIX architecture
+#endif
+
+#if defined(__GNUC__)
+#include <asm_inline_gcc.h> /* The empty one.. */
```

Comment:
Are this and next line referencing the same header?

Reply:
Nope, one is ./arch/posix/include/asm_inline.h , the other ./include/arch/posix/asm_inline.h.